builder/build.go

```diff
@@ -118,6 +118,15 @@ func buildFlagSlice(nocache bool, squash bool, httpProxy string, httpsProxy stri
 		spaceSafeBuildFlags = append(spaceSafeBuildFlags, "--build-arg", fmt.Sprintf("%s=%s", k, v))
 	}
 
+	if buildOption != "" {
+		buildOptionPackages, err := getBuildOptionPackages(buildOption, language)
+
+		//TODO is this ok?
+		if err == nil {
+			spaceSafeBuildFlags = append(spaceSafeBuildFlags, "--build-arg", fmt.Sprintf("ADDITIONAL_PACKAGE=%s", buildOptionPackages))
```

Comment:
We need to handle situations where the user has passed a `--build-arg` with the same name.